### PR TITLE
Update to ember-cli 2.4.1

### PR DESCRIPTION
**Pull request by Panman8201**
Updated ember-cli to version 2.4.1 and also ran `ember init` (only applying changes where they made sense). Finally added an ember-try scenario for ember 2.4.1.

### CODE REVIEW

File: testem.js [hunk 1]
```diff
@@ -1,11 +1,13 @@
-{
+/*jshint node:true*/
+module.exports = {
   "framework": "qunit",
   "test_page": "tests/index.html?hidepassed",
   "disable_watching": true,
   "launch_in_ci": [
-    "Firefox",
-    "Chrome"
+    "PhantomJS"
```
Comment (martndemus) [inline testem.js]: Pease remove Phantom and put back Firefox and Chrome.

File: testem.js [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
   ],
   "launch_in_dev": [
+    "PhantomJS",
+    "Chrome"
```
Comment (martndemus) [inline testem.js]: Same here.

File: tests/dummy/app/index.html [hunk 3]
```diff
@@ -3,23 +3,23 @@
   <head>
     <meta charset="utf-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <title>ember-cli-font-awesome</title>
+    <title>Dummy</title>
```
Comment (martndemus) [inline tests/dummy/app/index.html]: can you put back the title?

**Comment by Panman8201** [comment]
@martndemus Alright, I've addressed all of your comments. :smile: Not sure how I missed the testem stuff...

**Comment by martndemus** [comment]
Thanks, looks good now!
@homu r+

**Pull request merged**